### ISSUE TITLE
Undefined variable $additional_field in hook_message_view

### ISSUE THREAD

**Issue by mccrodp** [Gizra/message#26]
On D8 branch in _message.api.php_, I presume the code in `hook_message_view` is an example and the hook function should not be called directly, but is `$additional_field` intentionally undefined?

```
function hook_message_view($message, $view_mode, $langcode) {
  $message->content['my_additional_field'] = array(
    '#markup' => $additional_field,
    '#weight' => 10,
    '#theme' => 'mymodule_my_additional_field',
  );
}
```

**Comment by RoySegall** [comment]
I think most of the hooks are not relevant any more but on the other hand there are new hooks.

**Comment by mccrodp** [comment]
Ok, I see. I was doing some D7 work today and noticed that this was an issue there as well.

So, with the unused hooks and similar upgrade issues, is there a POA? Should I ignore, ensure / test they are unused, file bug reports on them?

**Comment by RoySegall** [comment]
POA? If something wrong, from you point of view, you can suggest a PR or elaborate here before going in to work. Contribution is always more then welcome :)

**Comment by mccrodp** [comment]
Sorry, I didn't elaborate; I mean, if the hooks are no longer used, should they be removed? For instance in this case, should I remove them and similar items after verifying they are not used, then suggest the PR?

**Comment by RoySegall** [comment]
Yeah, sure. If the hooks listed in the D8 branch then you can remove them and maybe improve the documentation. I pretty sure couple of them need to be removed and for other we need to alter the structure or the function's signature.

**Comment by mccrodp** [comment]
Ok, great, starting to look properly at the D8 upgrade of Message UI today, so I may come across things like this in Message. Thanks.

**Comment by RoySegall** [comment]
No, thank you!
